sparse_strips/vello_common/src/paint.rs

```diff
@@ -99,26 +99,37 @@ impl Image {
     /// # Panics
     ///
     /// This panics if `image` has a `width` or `height` greater than `u16::MAX`.
-    pub fn from_peniko_image(image: &peniko::Image) -> Self {
+    pub fn from_peniko_image(brush: &peniko::ImageBrush) -> Self {
         // TODO: how do we deal with `peniko::ImageFormat` growing? See also
         // <https://github.com/linebender/vello/pull/996#discussion_r2080510863>.
-        if image.format != peniko::ImageFormat::Rgba8 {
-            unimplemented!("Unsupported image format: {:?}", image.format);
+        if brush.image.format != peniko::ImageFormat::Rgba8 {
+            unimplemented!("Unsupported image format: {:?}", brush.image.format);
+        }
+        if brush.image.alpha_type != peniko::ImageAlphaType::AlphaPremultiplied {
```

Comment:
Actually vello_cpu natively support premultiplied, so we just need to repack same bytes to new type (well we still need to apply alpha).

Reply:
This PR is not going to make any behaviour differences (except for those due to the Kurbo update)